Gemfile

```diff
@@ -1,4 +0,0 @@
-source "https://rubygems.org"
```

Comment:
Does bundler just use the gem spec without a Gemfile? Or should we keep this, but just have `source` and `gemspec`?